config/query-api.yml

```diff
@@ -66,6 +66,28 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
+  /v1/webhook:
+    post:
+      summary: Data to add to RDS
+      description: Add data
+      operationId: query.lambdas.api_server.v1.endpoints.webhook
+      tags:
+        - All
+      parameters:
+        - name: new_data
+          in: body
+          schema:
+            type: string
```

Comment:
This is JSON right? I think that you might be able to get swagger to do [some validation](https://swagger.io/docs/specification/describing-parameters/) for you if you change the schema type to `object` and add some `properties`.

Reply:
👍